challenge-03_loops-conditionals/loops-conditionals/while/exercise_01.js

```diff
@@ -0,0 +1,16 @@
+//1. Create a while loop that will print out all the even numbers between 10 and 40.
+console.log("Even numbers between 10 and 40:");
+let evenNumber = 10;
+while (evenNumber <= 40) {
```

Comment:
These loops should iterate over all numbers in between 10 and 40, printing only the odd/even ones, depending on its own logic.

⚠️ FIX: iterate over all numbers and print only the odd/even ones